_posts/2018-10-23-artsy-auth.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Everything You Ever Wanted To Know About Authentication at Artsy (But Didn't Know How To Ask)
+date: 2018-10-23
+categories: [tech, oauth, security, infosec, authentication]
+author: myk
+---
+
+# Everything You Ever Wanted To Know About Authentication At Artsy (But Didn't Know How To Ask)
```

Comment:
This will be shown by our blog template based on the `title` front matter, so if we want to post it to the blog, we can remove this H1 👍